src/mmw/js/src/modeling/views.js

```diff
@@ -1086,6 +1090,22 @@ var ResultsDetailsView = Marionette.LayoutView.extend({
         this.panelsRegion.$el.show();
         this.contentRegion.$el.show();
     },
+
+    showSubbasinHuc12View: function(huc12Id) {
+        this.subbasinRegion.$el.hide();
+
+        this.subbasinHuc12Region.show(new SubbasinHuc12TabContentView({
+            model: this.collection.getResult('subbasin'),
+            scenario: this.scenario,
+            hideSubbasinHotSpotView: this.hideSubbasinHuc12View,
```

Comment:
Since `hideSubbasinHuc12View` uses `this` in its body, I think we may need to wrap this in a `_.bind()`. Haven't tested this out yet, although I'd be surprised if we don't need it.

Reply:
@kellyi made this a bit more like our react project and bound some methods up in init https://github.com/WikiWatershed/model-my-watershed/pull/2781/files/1eb4ff24059992c6b4e6cf4823c3896a9048121b#diff-416520c38e4c168e3f58d3f5089772c4R1040

I continued to follow that pattern, so `this.hideSubbasinHuc12View` is bound. Looking at that block again however, I'm unsure if it will work in IE. It makes use of `.bind` as a function built-in.

Reply:
I'll test IE.

Reply:
Tested in IE. Seems to be working. +1